go/config/config_test.go

```diff
@@ -0,0 +1,301 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func defaultServerConfig(t *testing.T) *ServerConfig {
+	return &ServerConfig{
+		Address:            "0.0.0.0:8080",
+		CertFile:           "",
+		KeyFile:            "",
+		CAFile:             "",
+		CORSAllowedOrigins: nil,
+	}
+}
+
+func userServerConfig(t *testing.T) *ServerConfig {
+	return &ServerConfig{
```

Comment:
Call t.Helper() here and everywhere else we have a testing helper function.

Reply:
Amended.